app/views/spaces/_sidebar_webconf_buttons.html.haml

```diff
@@ -19,7 +19,7 @@
         = link_to t('.end'), end_bigbluebutton_room_path(room, :redir_url => space_path(room.owner)), options_for_tooltip(t('.end_tooltip'), :data => { :confirm => t('are_you_sure') }, :class => 'btn btn-danger')
     - else
       -# only show the options dialog if the user can record and the record flag is not automatically set
-      - if can?(:record_meeting, room) && !current_site.webconf_auto_record
+      - if can?(:record_meeting, room)
```

Comment:
This `if` is not necessary too. The double buttons to start the conference should not appear anymore, it should only be a single button, as is done below in the `else`.

With this, there should be no more links to "join_options_*". This is the view that shows the "record" checkbox, that is not necessary anymore.